qi-test/tests/compiler/rules/full-cycle.rkt

```diff
@@ -39,9 +46,47 @@
     (test-true "normalize → deforest"
                (deforested?
                  (phase1-eval
-                  (test-compile~> #'(~>> (filter odd?) values (map sqr))
-                                  normalize-pass
-                                  deforest-pass)))))))
+                  (qi-compile
+                   #'(~>> (filter odd?) values (map sqr)))))))
+   (test-suite
+    "sandboxed evaluation"
+    (test-not-exn "Plays well with sandboxed evaluation"
```

Comment:
I'm starting to wonder if this test is too complicated: isn't the whole point just that `(require qi)` should work in a sandbox, it didn't before #168, and now it does? Why go through the hoops to generate a phase1-eval inside a sandboxed eval?

Reply:
🤦